types/autoprefixer/package.json

```
@@ -1,6 +1,6 @@
 {
     "private": true,
     "dependencies": {
-        "postcss": "7.x.x"
+        "postcss": "8.x.x"
```

what about explicit:
```suggestion
        "postcss": "^8.1.0"
```
https://github.com/postcss/autoprefixer/blob/master/package.json#L29